### DISCUSSION TITLE
webrtc Play address

### DISCUSSION THREAD

**Discussion by congkuan** [ant-media/Ant-Media-Server#5688] [FAQs]
I am trying to use a third-party player. How do I get the streaming address, which usually starts with webrt://

**Comment by lastpeony** [comment]
Which player do you use?
You need a websocket connection and messaging between client and AMS
Please have a look at this:

https://antmedia.io/docs/guides/publish-live-stream/webrtc/webrtc-websocket-messaging-reference/#playing-webrtc-stream
